frontend/src/export/mtga.js

```diff
@@ -0,0 +1,38 @@
+import { ZONE_MAIN, ZONE_SIDEBOARD } from "../zones";
+
+export default {
+  name: "MTG Arena",
+  // download,
+  // downloadExtension: "???"
```

Comment:
Could you put a note here to indicate why you commented these lines ?

Reply:
Yes!
Basically showing that in the future there could be download options of mtga starts storing a file format.

Reply:
See the note you put [here](https://github.com/dr4fters/dr4ft/pull/1419/files#diff-7c9a7fdc079f9f1076a8bc832bd135d03f2c65a5ca5ab0c285c47d751af55f52R7):
`  // copy // NOTE - mtgo does not support import from clipboard`

Something along the lines works here, too.